test/paper-date-picker.html

```diff
@@ -37,9 +37,10 @@
           // Check initialization
           expect(element.date).to.be.not.null;
           expect(element.maxDate).to.be.not.null;
+          expect(element.id).to.be.equal('picker');
 
           // Try to swipe after the max bound
-          var calendar = element.$$('paper-calendar');
+          var calendar = element.getCalendar();
```

Comment:
You can use `element.$.calendar` here to make this query, or you can query the shadowRoot directly as well instead of creating a function just for a test case.